python/cudf/cudf/core/series.py

```diff
@@ -823,7 +823,7 @@ def map(self, arg, na_action=None) -> "Series":
 
         Used for substituting each value in a Series with another value,
         that may be derived from a function, a ``dict`` or
-        a :class:`Series`.
+        a ``Series``.
```

Comment:
I just double checked this and you had this right with
```
:class:`Series`
```

Reply:
Great I'll change it back to that then!